### PR TITLE
tweaks Makefile and moves permissions there

### PR DESCRIPTION
**Pull request by kzidane**
Addresses https://github.com/cs50/python-cs50/issues/6!

cc: @dmalan

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,20 +1,22 @@
-BUILD_DIR = build
-DESCRIPTION = CS50 Library for Python
-MAINTAINER = CS50 <sysadmins@cs50.harvard.edu>
-NAME = python-cs50
-OLD_NAME = lib50-python
-VERSION = 1.2.4
+BUILD_DIR := build
+LIB_DIR := $(BUILD_DIR)/usr/lib
+PYTHON2_DIR := $(LIB_DIR)/python2.7/dist-packages/cs50
```
Comment (dmalan) [inline Makefile]: So we're (clearly) assuming Python 2.7 here. Is there a more dynamic way to figure out the right path on a system?

File: Makefile [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
 	docker run -i --rm -t -v "$(PWD)":/root cs50/cli
 
 .PHONY: build
 build: clean
-	mkdir -p "$(BUILD_DIR)"/usr/lib/python2.7/dist-packages/cs50
-	cp src/* "$(BUILD_DIR)"/usr/lib/python2.7/dist-packages/cs50
-	mkdir -p "$(BUILD_DIR)"/usr/lib/python3/dist-packages/cs50
-	cp src/* "$(BUILD_DIR)"/usr/lib/python3/dist-packages/cs50
+	mkdir -p "$(PYTHON2_DIR)" "$(PYTHON3_DIR)"
+	find "$(PYTHON2_DIR)" "$(PYTHON3_DIR)" -maxdepth 0 -exec cp src/* {} \;
```
Comment (dmalan) [inline Makefile]: Why this cleverness? Why not just `cp` twice?

File: Makefile (hunk 1 above)
Comment (kzidane) [inline Makefile]: Not sure yet! But even `sdist` (just emailed about) seems to assume this path, by default.

File: Makefile (hunk 2 above)
Comment (kzidane) [inline Makefile]: Just didn't want to write `src/*` twice, or factor it out, so replaced with this one-liner. Seems I've over-complicated? Might be subject to remove, if we're to transition to `sdist`, anyway!

**Comment by dmalan** [comment]
nbd just felt like an overoptimization!

**Pull request closed without merging**